test/commute.js

```diff
@@ -17,7 +16,26 @@ describe('commute', function() {
   });
 
   it('works on Algebraic Data Types such as "Maybe"', function() {
```

Comment:
[These](http://ramdajs.com/0.17/docs/#commute) examples in the docs. Replace them with `Future` instead so we have `Maybe` and `Future`. 

Just a suggestion...